dex-documentation/documentation.md

```diff
@@ -1,144 +1,144 @@
-- **SwapExactTokensForTokensOnXYKPool** - exchange, deposit A receive B, indicate desired amount for A.
-- **SwapTokensForExactTokensOnXYKPool** - exchange, deposit A receive B, indicate desired amount for B.
+*User ISIs:*
+- **AddLiquidityToXYKPool** - deposit tokens into pool: deposit A, deposit B, receive pool tokens.
+- **RemoveLiquidityFromXYKPool** - burn pool tokens, receive A and B.
+- **SwapExactTokensForTokensOnXYKPool** - exchange, deposit A and receive B, indicate desired amount for A.
+- **SwapTokensForExactTokensOnXYKPool** - exchange, deposit A and receive B, indicate desired amount for B.
 
-NOTE: Behavior for **RemoveTokenPair** and **RemoveLiquiditySource** is currently unspecified. Removal of associated entities should induce return of locked liquidity and removal of pool tokens from liquidity providers to avoid misuse if pool is re-initialized.
+NOTE: The behavior for **RemoveTokenPair** and **RemoveLiquiditySource** is currently unspecified. Removal of associated entities should induce return of locked liquidity and removal of pool tokens from liquidity providers to avoid misuse if the pool is re-initialized.
 
 *Queries:*
-- **GetDEX**, **GetDEXList** - get info about particular DEX, list initialized DEX'es.
-- **GetTokenPair**, **GetTokenPairList**, **GetTokenPairCount** - get info about TokenPairs.
-- **GetXYKPoolInfo** - get info about state of particular XYK Pool.
+- **GetDEX**, **GetDEXList** - get info about particular DEX, list initialized DEXes.
+- **GetTokenPair**, **GetTokenPairList**, **GetTokenPairCount** - get info about token pairs.
+- **GetXYKPoolInfo** - get info about state of particular XYK pool.
 - **GetFeeOnXYKPool**, **GetProtocolFeePartOnXYKPool** - get info about current fees.
 - **GetPriceForInputTokensOnXYKPool** - get price of output token, if exact input amount is specified.
 - **GetPriceForOutputTokensOnXYKPool** - get price of input token,  if exact output amount is specified.
-- **GetOwnedLiquidityOnXYKPool** - specified amount of pool tokens, get corresponding amounts of pair tokens.
+- **GetOwnedLiquidityOnXYKPool** - specify amount of pool tokens, get corresponding amount of pair tokens.
 
-*Debug ISI's*
+*Debug ISIs*
 - **AddTransferPermissionForAccount** - allow user to transfer own tokens.
 
 ## Code Structure
 
-Core logic of DEX is contained in a `dex` rust module (`dex.rs`) in `iroha` crate. That approach was done in order to keep iroha, which avoids being monolithic with all features, isolated from dex. Although some minor proposals were accepted into iroha core because of new functionality requirements, but they are of general use.
+The core logic of Iroha2 DEX is contained in a `dex` rust module (`dex.rs`) in the `iroha` crate. This approach was taken in order to keep Iroha Core isolated from DEX and prevent it from becoming monolithic in its features. Although some minor proposals were accepted into Iroha Core because of new functionality requirements, these are of general use.
```

Comment:
'which avoid being monolithic with all features' relates to current iroha, rather than approach taken. Fix does not contradict this, so no need to change I guess.